src/sources/ncdatasets.jl

```diff
@@ -48,20 +48,21 @@ const NCD_FILL_TYPES = (Int8,UInt8,Int16,UInt16,Int32,UInt32,Int64,UInt64,Float3
 
 # CF standards don't enforce dimension names.
 # But these are common, and should take care of most dims.
-const DIMMAP = Dict("lat" => Lat,
-                    "latitude" => Lat,
-                    "lon" => Lon,
-                    "long" => Lon,
-                    "longitude" => Lon,
-                    "time" => Ti,
-                    "lev" => Vert,
-                    "mlev" => Vert,
-                    "level" => Vert,
-                    "vertical" => Vert,
-                    "x" => X,
-                    "y" => Y,
-                    "z" => Z,
-                   )
+const DIMMAP = Dict(
+    "lat" => Y,
+    "latitude" => Y,
+    "lon" => X,
+    "long" => X,
+    "longitude" => X,
+    "time" => Ti,
+    "lev" => Z,
+    "mlev" => Z,
+    "level" => Z,
+    "vertical" => Z,
+    "x" => X,
+    "y" => Y,
+    "z" => Z,
```

Comment:
Will this handle capitalized dim names? Maybe you're already doing a "lowercase" somewhere, or maybe dims are already usually lowercase? @rafaqz